uiowa/uiowa
# Update P2LB block default styles

Resolves #5317 

# How to test

`ddev blt ds --site=hawkeyemarchingband.uiowa.edu`
`ddev drush @hawkeyemarchingband.local en sitenow_p2lb`
`ddev drush @hawkeyemarchingband.local config-split:activate p2lb`
Go to `https://hawkeyemarchingband.uiowa.ddev.site/admin/content` and select a page to convert to p2lb.

## How to test style updates
- Convert a page with cards
  - Verify that subtitle field is rendered within the subtitle card block field and not the body.
  - Verify that the text format is rendered as `minimal` and that the buttons in the blocks on https://hawkeyemarchingband.uiowa.ddev.site/ render as links and not styled buttons. 
  - Verify that the card is converted to a block that has the `card: border` style applied to it. 
## How to test blockquote
- Create a page with a series of blockquote paragraphs and convert it (blockquotes within the WYSIWYG will not be converted). 
  - Verify that the quotes convert to the blockquote block type and not into a text area. 
  - Verify that the converted quote style is left-aligned.
  - Add your opinion on this question on the blockquote alignment options: https://github.com/uiowa/uiowa/blob/3f2a67c01819546713650377ed489c336f619b2d/docroot/modules/custom/sitenow_p2lb/sitenow_p2lb.module#L474

## How to test more paths

Either the v2 split needs to then be disabled, or the following code needs to be updated locally to avoid the v2 check: https://github.com/uiowa/uiowa/blob/main/docroot/modules/custom/layout_builder_custom/layout_builder_custom.module#L1240 (eg manually set $is_v2 to FALSE before the check). In either case, a `drush cr` is likely needed after as well.

- Convert a page with Articles and People paragraphs and check:
  - The 'include more link' converts correctly
  - If a link is included but no path is provided, the default more path is properly used
  - If a link is included and a path is provided, that this path replaces the default path
